user_guide_src/source/libraries/validation.rst

```diff
@@ -972,10 +972,13 @@ regex_match             Yes        Fails if field does not match the regular
                                    expression.
 required                No         Fails if the field is an empty array, empty
                                    string, null or false.
+required_if             Yes        The field is required when any of the other    ``required_if[field,value1,value2,...]``
+                                   field has the given value. (This rule was 
+                                   added in v4.6.0.)
```

Comment:
This explanation is not correct. There is only one other field.

> The field is required when the other field has the given value(s).

Reply:
done, updated.